client/signup/controller.js

```diff
@@ -263,6 +265,45 @@ export default {
 			context.store.dispatch( setSelectedSiteId( null ) );
 		}
 
+		if ( flowName === 'onboarding' || flowName === 'onboarding-plan-first' ) {
+			context.primary = (
+				<Experiment name="domain_step_plan_step_swap">
```

Comment:
I started the experiment without this PR being merged, so we'll have to change this experiment name. How about `signup_domain_plan_step_swap`?